Kernel/Arch/x86_64/Interrupts.cpp

```diff
@@ -173,41 +118,19 @@ static EntropySource s_entropy_source_interrupts { EntropySource::Static::Interr
             "    jmp common_trap_exit \n"                                      \
         );                                                                     \
     }
-#endif
 
 // clang-format on
 
 void dump_registers(RegisterState const& regs)
 {
-#if ARCH(I386)
-    u16 ss;
-    u32 esp;
-
-    if (!(regs.cs & 3)) {
-        ss = regs.ss;
-        esp = regs.esp;
-    } else {
-        ss = regs.userspace_ss;
-        esp = regs.userspace_esp;
-    }
-#else
     u64 rsp;
 
     if (!(regs.cs & 3))
         rsp = regs.rsp;
     else
         rsp = regs.userspace_rsp;
-#endif
 
     dbgln("Exception code: {:04x} (isr: {:04x})", regs.exception_code, regs.isr_number);
-#if ARCH(I386)
-    dbgln("    pc={:#04x}:{:p} eflags={:p}", (u16)regs.cs, regs.eip, regs.eflags);
-    dbgln(" stack={:#04x}:{:p}", ss, esp);
-    dbgln("    ds={:#04x} es={:#04x} fs={:#04x} gs={:#04x}", (u16)regs.ds, (u16)regs.es, (u16)regs.fs, (u16)regs.gs);
-    dbgln("   eax={:p} ebx={:p} ecx={:p} edx={:p}", regs.eax, regs.ebx, regs.ecx, regs.edx);
-    dbgln("   ebp={:p} esp={:p} esi={:p} edi={:p}", regs.ebp, regs.esp, regs.esi, regs.edi);
-    dbgln("   cr0={:p} cr2={:p} cr3={:p} cr4={:p}", read_cr0(), read_cr2(), read_cr3(), read_cr4());
-#else
```

Comment:
could we turn that into `#if ARCH(X86_64)` with a ‘to doʼ for `ARCH(AARCH64)`? 🫢